modules/effects/arrow-effects/src/main/kotlin/arrow/effects/typeclasses/CancellableEffect.kt

```diff
@@ -0,0 +1,8 @@
+package arrow.effects.typeclasses
+
+import arrow.Kind
+import arrow.core.Either
+
+interface CancellableEffect<F> : Effect<F> {
```

Comment:
I believe this should be just `Concurrent` and include the `race` style algebra combinators in this type class so they are available for all concurrent effects in the spirit that concurrent effects are cancelable. For what is worth in cats we have:
```
Concurrent[F] extends Async[F]
ConcurrentEffect[F] extends Concurrent[F] with Effect[F]
```